CHANGELOG.md

```diff
@@ -52,6 +52,22 @@ All notable changes to this project will be documented in this file based on the
       "number_of_significant_value_digits" : 3
     }
   ```
+
+### Added
+
+* Added a transport class for mocking a HTTP 403 error codes, useful for testing response failures in inheriting clients
```

Comment:
@maltehuebner You don't seem to have luck with the changelog and rebase :-(

Reply:
Next try. I’m sure I’ll never get a job again if someone sees my git merging skills.

Reply:
No worries, it's not you. It's the 3-way merge commit magic from Git that soemtimes does funny stuff here :-)